lib/config.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+
+let _       = require('lodash'),
+    path    = require('path');
+
+class Config {
+
+    constructor(pkg, options, basePath) {
+
+        this.options = options;
+        this.basePath = basePath;
+        this.cacheControlHeader = process.env.CACHE_CONTROL || this.options.maxAge || 'max-age=60';
+        this.customHeaders = this.options.customHeaders || [];
+        this.description = this.options.description || pkg.description;
+        this.env = process.env.ENVIRONMENT || process.env.NODE_ENV || this.options.host || 'development';
+        this.envProd = this.options.envProd || 'production';
+        this.port = process.env.PORT || this.options.port || 3000;
+        this.host = process.env.HOST || this.options.host || '0.0.0.0';
```

Comment:
why not just set the port to the environment in the options...? As it is in the example app. e.g. 
```js 
let server = hapi.instance({ port: process.env.PORT });
```
or don't set the option and let it default to the environment var.